code/edgebridge/build.gradle

```diff
@@ -208,8 +179,9 @@ task platformUnitTestJacocoReport(type: JacocoReport, dependsOn: "testPhoneDebug
     additionalClassDirs.setFrom files([debugTree])
     additionalSourceDirs.setFrom files(android.sourceSets.main.java.sourceFiles)
     sourceDirectories.setFrom files(android.sourceSets.phone.java.sourceFiles)
-    executionData "$buildDir/jacoco/testPhoneDebugUnitTest.exec"
-
+//    executionData "$buildDir/jacoco/testPhoneDebugUnitTest.exec"
+    executionData "$buildDir/outputs/unit_test_code_coverage/phoneDebugUnitTest/testPhoneDebugUnitTest.exec"
+//    /Users/timothyk/git/timkimadobe/aepsdk-edgebridge-android/code/edgebridge/build/outputs/unit_test_code_coverage/phoneDebugUnitTest
```

Comment:
Is this creating the coverage reports for you?  I noticed the Core project is using this path as well.
Please remove the commented out lines of code.